### ISSUE TITLE
Softer foreground whites

### ISSUE THREAD

**Issue by thomashoggard** [Monokai/monokai-pro-vscode#81]
Is it possible to soften the whites so they aren't as bright?  I personally find themes with a more slightly grey font helps reduce eye strain.

**Comment by dws-git** [comment]
Agreed. Not much, just slightly off-white would be much easier on the eyes.

**Comment by mrmivo** [comment]
I second this. I'd already have bought the theme if not for the bright white.

**Comment by 7RamzeS7** [comment]
Please make the white font less bright. I use the theme: Monokai Pro (filter Spectrum) and in this theme the white font is very bright (ripples in the eyes) or add the ability to customize.

**Comment by Monokai** [comment]
If I would dim the white by default, I would probably get requests to make it brighter, because for some the contrast would then be too low or the white too dark.

Most laptops and monitors have a brightness setting. I think it's best to use that, if you think the white is too bright. Usually when it's darker, I'm decreasing the brightness of my screen (or it happens automatically), and it all works out nicely.

**Comment by SimWhite** [comment]
> If I would dim the white by default, I would probably get requests to make it brighter, because for some the contrast would then be too low or the white too dark.
Are there any ways to change it using `editor.tokenColorCustomizations` for example? I want to change foreground text color from #fcfcfa to #cccccc.

**Comment by anthonypenna** [comment]
Agreed, the pure white foreground strains my eyes, would be nice to dim it down a little bit.

**Comment by SimWhite** [comment]
I changed everything from `#fcfcfa` to `#cccccc` at `~/.vscode/extensions/monokai.theme-monokai-pro-vscode-1.1.13/themes/Monokai Pro.json` and it looks good now.

**Comment by pkazmier** [comment]
Yes, I really notice this “issue” with markdown files. It’s blistering white on the eyes. I just added this block to my prefs file and the improvement was considerable in my opinion.

```json
    "editor.tokenColorCustomizations": {
        "[Monokai Pro]": {
            "textMateRules": [
                {
                    "scope": "markup.inline.raw.string.markdown",
                    "settings": { "foreground": "#fc9867" }
                },
                {
                    "scope": "meta.paragraph.markdown",
                    "settings": { "foreground": "#c1c0c0" }
                }
            ]
        }
    }
```

Now I’m on a mission to tweak the whites elsewhere as needed, such as the integrated terminal. I’m mixed on the whites in, say, a Python file as they are not in large quantities strung together in long sequences.

I know theming is subjective, but my eye strain is not :-)

**Comment by pkazmier** [comment]
Interestingly, I noticed that I prefer the theme as designed when viewed on my iMac Pro with Mojave. The "whites" looks very good, so please disregard my previous comment on this issue. I think it has to do with Apple disabling sub-pixel anti-aliasing as the characters are not as "thick" on Mojave as they were on High Sierra. As a result, the brightness of the white is subdued because the letterforms are thinner and thus fewer pixels are white.

So, long story short, Monokai Pro looks great on Mojave! I also just purchased it today as I know how much time and effort goes into hacking themes.

**Comment by tracphil** [comment]
I agree. The whites in Sublime Text are blinding. I tried it in VSCode just to see what it looked like there and it was ok.

Is there a way you can provide a setting in Sublime Text to dim the white fonts? At least it is optional end not forced on the folk that don't want it.
